packages/baxter/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/djunekz/baxter
+TERMUX_PKG_DESCRIPTION="Simple Encrypt and Decrypt file bash"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=v1.2.2                                                                                                                               TERMUX_PKG_SRCURL=https://github.com/djunekz/baxter/archive/${TERMUX_PKG_VERSION}/baxter-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=d72db514661220cf76a2aa9c71f8ebc406a5955debc279b687b9fbcb40da48bb                                                                      TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
There should be newlines between these values, this was likely caused by an editor problem.
```suggestion
TERMUX_PKG_SHA256=d72db514661220cf76a2aa9c71f8ebc406a5955debc279b687b9fbcb40da48bb
TERMUX_PKG_AUTO_UPDATE=true
```